narwhals/_pandas_like/namespace.py

```diff
@@ -108,6 +108,30 @@ def all(self) -> PandasExpr:
             implementation=self._implementation,
         )
 
+    def lit(self, value: Any, dtype: dtypes.DType | None) -> PandasExpr:
+        def _lit_pandas_series(df: PandasDataFrame) -> PandasSeries:
+            pandas_series = PandasSeries(
+                series_from_iterable(
+                    [value for _ in range(len(df._dataframe.index))],
+                    name="lit",
+                    index=df._dataframe.index,
```

Comment:
can you just do
```suggestion
                    [value],
                    name="lit",
                    index=df._dataframe.index[0:1],
```
so the `validate_series_comparand` / `validate_dataframe_comparand` can take care of broadcasting? I'd expect it to be more efficient than allocating `len(df)` copies of `value`

Reply:
Thanks! I was expeciting there was a more efficient way to do this. 😅

Stupid question: are these `validate_series_comparand` / `validate_dataframe_comparand` implemented by the various libraries?

Reply:
had to revert this for "list" literal value 😕 please let me know if there is a better way